docs/02.API-REFERENCE.md

```diff
@@ -305,11 +305,14 @@ when the snapshot is generated and executed. Furthermore the
 
 ## jerry_exec_snapshot_opts_t
 
-Flags for [jerry_exec_snapshot](#jerry_exec_snapshot) and
-[jerry_load_function_snapshot](#jerry_load_function_snapshot) functions:
+Flags for [jerry_exec_snapshot](#jerry_exec_snapshot) functions:
 
  - JERRY_SNAPSHOT_EXEC_COPY_DATA - copy snapshot data into memory (see below)
  - JERRY_SNAPSHOT_EXEC_ALLOW_STATIC - allow executing static snapshots
+ - JERRY_SNAPSHOT_EXEC_LOAD_AS_FUNCTION - load snapshot as function instead of executing it
+
+*Changed in version [[NEXT_RELEASE]]*: The `JERRY_SNAPSHOT_EXEC_LOAD_AS_FUNCTION` value is added,
+                                       which replaces the `jerry_load_function_snapshot_at` function.
```

Comment:
* There is no "jerry_load_function_snapshot_at" function.
* This new option should be clearly mentioned in the "jerry_exec_snapshot" docs as with this we are merging two functions (also we need a relevant example).